src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4173,7 +4142,68 @@ namespace ZoneEquipmentManager {
 
         switch (thisZEqList.LoadDistScheme) {
         case DataZoneEquipment::LoadDist::SequentialLoading:
-            // Nothing to do here for this case
+            {
```

Comment:
The changes in here seem reasonable, but real evidence would come from test file and unit test results.